pdstemplate/__init__.py

```
@@ -1125,6 +1129,13 @@ def FILE_TIME(filepath):
             `filepath` in the form "yyyy-mm-ddThh:mm:ss".
         """
 
+        ##################################################################################
+        logger = get_logger()
+        logger.warning(f'FILE_TIME reflects the cached time: {filepath}', force=True)
```

Ruff (0.14.8)

Logging statement uses f-string

(G004)